.agents/architecture.md

```
@@ -972,15 +978,16 @@ When `@vuecs/theme-tailwind` is loaded:
 1. bg-primary-600          ← Tailwind v4 utility class
 2. --color-primary-600     ← @theme mapping in theme-tailwind/assets/index.css
 3. --vc-color-primary-600  ← semantic-scale var (overridden by setColorPalette)
-4. --color-blue-600        ← theme-tailwind rebind to Tailwind palette
+4. --color-blue-600        ← design/assets/index.css references the palette catalog
 5. oklch(...)              ← concrete OKLCH literal (Tailwind palette source)
 ```
 
-When only `@vuecs/design` is loaded (BS / Bulma / no theme):
+When `@vuecs/design/standalone` is loaded (BS / Bulma):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)